dispatchr-ruby/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,77 @@
+class UsersController < ApplicationController
+
+  def index
+    render json: User.all
+  end
+
+  def show
+    if params[:id].present?
+      user_id = params[:id]
+      user = User.find(user_id)
+
+      if user.present?
```

Comment:
User.find will respond with a 404 error on its own if no User is found. So you don't need to check if its present.